spec/functional/command/init_spec.rb

```diff
@@ -102,8 +102,8 @@ module Pod
           # platform :ios, '9.0'
 
           target 'App' do
-            # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
-            # use_frameworks!
+            # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
```

Comment:
why this change?

Reply:
From the changes i picked up [here](https://github.com/CocoaPods/Xcodeproj/pull/611) had to update the tests